Makefile

```diff
@@ -11,6 +11,8 @@ lint:
 	flake8 . && gulp lint --production
 
 test: lint
```

Comment:
always :) it's just that the PR was diffed against master and i was waiting on the other review :)